app/bundles/PageBundle/Form/Type/PageListType.php

```diff
@@ -57,21 +60,25 @@ public function configureOptions(OptionsResolver $resolver)
                         $choices[$page['language']][$page['id']] = "{$page['title']} ({$page['id']})";
                     }
 
+                    $choices = array_flip($choices);
```

Comment:
This is not correct. I bet it will even throw an error as array_flip can flip only array with strings or ints as keys and values. In this case the value is an array. Flip it in the foreach above instead.

See https://www.php.net/manual/en/function.array-flip.php